components/radio/stories/template.js

```diff
@@ -57,6 +62,12 @@ export const Template = ({
 					// Make checked value immutable for read-only.
 					e.preventDefault();
 				}}
+				@focusin=${function() {
+					updateArgs({ isFocused: true });
+				}}
+				@focusout=${function() {
+					updateArgs({ isFocused: false });
+				}}
```

Comment:
```suggestion
```
I'd recommend removing this, as it will show the focus ring on click in the Default story. It should only show on tab focus. That's because these events are different behavior than the `:focus-visible` in our styles.